exporter/api/server.go

```diff
@@ -71,7 +71,12 @@ func (ws *wsServer) Start(addr string) error {
 		zap.String("addr", addr),
 		zap.Strings("endPoints", []string{"/query", "/stream"}))
 
-	err := http.ListenAndServe(addr, ws.router)
+	server := http.Server{
+		Addr:              addr,
```

Comment:
@nkryuchkov what is this change for?

Reply:
@nivBlox Alternatively, I can add a `// nolint: gosec` comment if you'd prefer

Reply:
please add nolint and open a task on @amir-blox

Reply:
@nivBlox done